kvm/capabilities.go

```diff
@@ -0,0 +1,461 @@
+	CapCoalescedPIO             Capability = 162
+	CapHyperVEnlightenedVMCS    Capability = 163
+	CapExceptionPayload         Capability = 164
+	CapARMVMIPASize             Capability = 165
+	CapManualDirtyLogProtect    Capability = 166 /* Obsolete */
+	CapHyerVCPUID               Capability = 167
+	CapManualDirtyLogProtect2   Capability = 168
+	CapPPCIRQXive               Capability = 169
+	CapARMSVE                   Capability = 170
+	CapARMPTRAuthAddress        Capability = 171
+	CapARMPTRAuthGeneric        Capability = 172
+	CapPMUEventFilter           Capability = 173
+	CapARMIRQLineLayout2        Capability = 174
+	CapHyperVDirectTLBFlush     Capability = 175
+	CapPPCGuestDebugSStep       Capability = 176
+	CapARMNISVToUser            Capability = 177
+	CapARMInjectEXTDABT         Capability = 178
+	CapS390VCPUResets           Capability = 179
+	CapS390Protected            Capability = 180
+	CapPPCSecureGuest           Capability = 181
+	CapHALTPoll                 Capability = 182
+	CapASYNCPFInt               Capability = 183
+	CapLastCPU                  Capability = 184
+	CapSmallerMaxPhyAddr        Capability = 185
+	CapS390DIAG318              Capability = 186
+	CapStealTime                Capability = 187
+	CapX86UserSpaceMSR          Capability = 188
+	CapX86MSRFilter             Capability = 189
+	CapEnforcePVFeatureCPUID    Capability = 190
+	CapSysHyperVCPUID           Capability = 191
+	CapDirtyLogRing             Capability = 192
+	CapX86BusLockExit           Capability = 193
+	CapPPCDAWR1                 Capability = 194
+	CapSetGuestDebug2           Capability = 195
+	CapSGXAttribute             Capability = 196
+	CapVMCopyEncContextFrom     Capability = 197
+	CapPTPKVM                   Capability = 198
+	CapHyperVEnforceCPUID       Capability = 199
+	CapSREGS2                   Capability = 200
+	CapEXitHyperCall            Capability = 201
+	CapPPCRPTInvalidate         Capability = 202
+	CapBinaryStatsFD            Capability = 203
+	CapExitOnEmulationFailure   Capability = 204
+	CapARMMTE                   Capability = 205
+	CapVMMoveEncContextFrom     Capability = 206
+	CapVMGPABits                Capability = 207
+	CapXSave2                   Capability = 208
+	CapSysAttributes            Capability = 209
+	CapPPCAILMode3              Capability = 210
+	CapS390MemOpExtention       Capability = 211
+	CapPMUCapability            Capability = 212
+	CapDisableQuirks2           Capability = 213
+	CapVMTSCControl             Capability = 214
+	CapSystemEventData          Capability = 215
+	CapARMSystemSuspend         Capability = 216
+	CapS390ProtectedDump        Capability = 217
+	CapX86TripleFaultEvent      Capability = 218
+	CapX86NotifyVMExit          Capability = 219
+	CapVMDisableNXHugePages     Capability = 220
+	CapS390ZPCIOP               Capability = 221
+	CapS390CPUTOPOLOGY          Capability = 222
+	CapDirtyLogRingACQRel       Capability = 223
+)
+
+func (c Capability) String() string {
```

Comment:
I prefer automatic code generation. Could you please use this tool?

https://pkg.go.dev/golang.org/x/tools/cmd/stringer
https://github.com/bobuhiro11/gokvm/blob/main/kvm/exittype_string.go

Reply:
Sure do